server/liveblog/blogs/blogs.py

```diff
@@ -108,6 +109,17 @@ def send_email_to_added_members(blog, recipients, blog_url):
 class BlogService(BaseService):
     notification_key = "blog"
 
+    def get(self, req, lookup):
+        if req is None:
+            req = ParsedRequest()
+
+        if req.where:
+            docs = super().get_from_mongo(req, lookup)
+        else:
+            docs = super().get(req, lookup)
+
+        return docs
```

Comment:
nit?:
```suggestion
    def get(self, req, lookup):
        req = req or ParsedRequest()
        method = super().get_from_mongo if req.where else super().get
        return method(req, lookup)
```

also, can you please add some docstrings just to clarify the reason of this override?

Reply:
Sure.